document/FileFormat.mdk

```diff
@@ -2696,31 +2703,318 @@ The tool might provide a "related location" (&sect;[#result-relatedLocations]) w
+|                  | fully qualified name of     |                             |                             |
+|                  | the function being          |                             |                             |
+|                  | entered.                    |                             |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `functionExit`   | Recommended: The            | Optional: If the function   | Absent                      |
+|                  | fully qualified name of     | returns a value or          |                             |
+|                  | the function being          | values, in which case,      |                             |
+|                  | returned from.              | string representations      |                             |
+|                  |                             | of the value or values      |                             |
+|                  |                             | returned.                   |                             |
+|                  |                             | Absent: If the function     |                             |
+|                  |                             | does not return a value.    |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `functionReturn` | Recommended: The            | Optional: If the function   | Optional: If the function   |
+|                  | fully qualified name of     | returns a value or          | returns a value or          |
+|                  | the function being          | values, in which case,      | values, in which case,      |
+|                  | exited.                     | string representations      | the names and values        |
+|                  |                             | of the value or values      | of any variables            | 
+|                  |                             | returned.                   | participating in the        |
+|                  |                             | Absent: If the function     | return expression           |
+|                  |                             | does not return a value.    | or expressions.             |
+|                  |                             |                             | Absent: If the function     |
+|                  |                             |                             | does not return a value.    |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `usage`          | Recommended: The            | Optional: A string          | Absent                      |
+|                  | name of the variable        | representation of the       |                             |
```

Comment:
Fixed.

---
In reply to: [77425882](https://github.com/sarif-standard/sarif-spec/pull/264#discussion_r77425882) [](ancestors = 77425882)